box.json

```diff
@@ -14,7 +14,7 @@
       "in": "lib"
     },
     {
-      "name": "{\\.[php]}",
+      "name": "{\\.[php|bash|fish|zsh]}",
```

Comment:
this should probably be `\\.(php|bash|fish|zsh)` where did the square brackets come from?